msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -209,6 +213,19 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office app for Android
+
+To view your app running in Office app for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
+1. Select the **Apps** icon. Your sideloaded app appears among installed apps.
```

Comment:
@erikadoyle @mobajemu-msft  user needs to restart Office app to start seeing the sideloaded apps.

Reply:
@sheenamakker this is an earlier version. The latest that just got merged has this note.